pkg/server/api.go

```diff
@@ -169,7 +176,7 @@ func (sh *APIHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		serveConf = &converted31
 	} else {
 		// Can only be 2.2 here
-		converted2, err := ctrlcommon.ConvertRawExtIgnitionToV2(conf)
+		converted2, err := ctrlcommon.ConvertRawExtIgnitionToV2_2(conf)
```

Comment:
That was just me changing the name to match the others while I was in there since the other ones for V3 all have it specified. I was thinking of the poor soul that might have to do the next ignition bump. 

It's still the same V2 version (2.2)  it ever was (and hopefully the only version that ever will be)